src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ItemsRepeaterScrollHost.cs

```diff
@@ -274,7 +274,7 @@ internal void StartBringIntoView(UIElement element, double alignmentX, double al
 		void IRepeaterScrollingSurface.RegisterAnchorCandidate(UIElement element) => RegisterAnchorCandidate(element);
 		internal void RegisterAnchorCandidate(UIElement element)
 		{
-			if (HorizontalAnchorRatio != double.NaN || VerticalAnchorRatio != double.NaN)
+			if (!double.IsNaN(HorizontalAnchorRatio) || !double.IsNaN(VerticalAnchorRatio))
```

Comment:
I'm not aware of any bugs this can be causing currently. If someone knows, let me know to add a test and to close any existing issues for it (if any).

Reply:
/cc @dr1rrb

@Youssef1313 did you find this through an analyzer?

Reply:
Yes. https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2242